sinatra/skillsengine.rb

```diff
@@ -0,0 +1,36 @@
+require 'rest-client'
+require 'json'
+
+module SkillsEngine
+  @@client_id = ENV['SKILLS_ENGINE_ID']
+  @@client_secret = ENV['SKILLS_ENGINE_SECRET']
+  @@access_token = nil
+
+  private_class_method def self.get_access_token()
```

Comment:
This can be named `access_token`, then you can just use it in the other method like a variable. It won't refer to the class variable unless it's prepended by `@@`. Helps with the `get_access_token` vs `fetch_access_token` naming too.